content/core-features/authentication.textile

```diff
@@ -373,6 +374,125 @@ The following JavaScript example demonstrates how to issue an "Ably Token":#ably
   })
 ```
 
+h2(#token-revocation). Token revocation
+
+Token revocation is a mechanism that enables an app to invalidate authentication tokens. This invalidation can be used to force specified clients to re-obtain a token, and subsequently enables the app to modify the rights granted to clients, or to decline to re-issue a token.
+
+The main use case of token revocation is as a method to combat malicious clients. In cases where the client behaves safely, you don't need token revocation - your code can instruct the connection to Ably to close (or detach from a channel it shouldn't have access to). But if you want to prevent any chance of a user obtaining the token before it is disconnected, and using it in their own custom client to gain access to Ably, then token revocation can be used.
+
+Without token revocation, in order to restrict potential rogue access, you would make tokens with a short TTL (say 10 minutes) and then simply not issue a new token to a suspect client. Unfortunately, reducing the TTL would increase load on your authentication server, so token revocation can provide a more scalable solution to the issue of rogue clients.
+
+Token revocation has to be enabled for you by Ably. There is a performance cost to enabling this feature. "Contact Ably":https://ably.com/contact if you'd like to enable this feature for a specific application.
+
+h3. Revocable tokens
+
+Tokens are revocable if they are created from an API key that has a @revocable@ attribute set to @true@. 
+
+Tokens created from a key with the @revocable@ attribute set to @true@ are restricted to a TTL of 1 hour; an attempt to create, or present, a token that is revocable, but has an TTL of longer than 1 hr, is rejected with an error.
+
+This functionality applies to client-signed JWTs and Ably-issued tokens, the latter being tokens obtained using the "@requestToken()@ API":/realtime/authentication#request-token).
```

Comment:
This paragraph seems unnecessary -- all tokens are either JWTs or ably-issued tokens, so it's not excluding anything. (the way it's worded might make someone think that it doesn't apply if you're using [the auth flow where](https://ably.com/documentation/core-features/authentication#token-request-process) you supply a tokenrequest to a client and it turns that into a token? but that works fine, because the client just turns it into a token).

Reply:
Removed.